lib/pageflow/configuration.rb

```diff
@@ -266,6 +266,11 @@ class Configuration
     # @since edge
     attr_reader :permissions
 
+    # Defines the editor lock polling interval.
+    # @return [number]
+    # @since 0.12
+    attr_reader :editor_lock_interval
```

Comment:
It'c called "edit lock" in the rest of the code. Let's stick with that. Can you still fix the `@since` tag?